happytransformer/tc/trainer.py

```diff
@@ -40,13 +40,10 @@ def test(self, input_filepath, solve):
         """
         contexts = self._get_data(input_filepath, test_data=True)
 
-        results = list()
-
-        for context in tqdm(contexts):
-            result = solve(context)
-            results.append(result)
-
-        return results
+        return [
+            solve(context)
+            for context in tqdm(contexts)
```

Comment:
Also a little awkward

Reply:
I disagree. This is a perfect case for a list comprehension

Reply:
ditto